### PR TITLE
Change provider logos to plain text

### PR DESCRIPTION
**Pull request by dreampulse**
resolves https://github.com/all3dp/printing-engine-client/issues/1308

### CODE REVIEW

**Comment by webholics** [comment]
PixelSnap 2019-08-28 at 14 49 07

This doesn't look optimal. The bold black type for the provider in my opinion doesn't fit well to the other content in the card. Also there is a spacing above the provider name which looks odd now.

File: src/app/component/cart-model-item/cart-model-item.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ CartModelItem.propTypes = {
   price: PropTypes.string,
   info: PropTypes.node,
   shippingInfo: PropTypes.node,
-  providerImage: PropTypes.element,
+  provider: PropTypes.node,
```
Comment (webholics) [inline src/app/component/cart-model-item/cart-model-item.js]: is should still be `PropTypes.element`

File: src/app/component/cart-model-item/cart-model-item.story.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ import CartModelItem from '.'
 import Button from '../button'
 import ButtonBar from '../button-bar'
 import NumberField from '../number-field'
-import ProviderImage from '../provider-image'
 
 import placeholderIcon from '../../../asset/icon/placeholder.svg'
+import ProviderName from '../provider-name'
```
Comment (webholics) [inline src/app/component/cart-model-item/cart-model-item.story.js]: component imports should be placed together

File: src/app/component/model-list/model-list.story.js [hunk 3]
```diff
@@ -7,12 +7,12 @@ import Button from '../button'
 import ButtonBar from '../button-bar'
 import CartModelItem from '../cart-model-item'
 import NumberField from '../number-field'
-import ProviderImage from '../provider-image'
 
 import placeholderIcon from '../../../asset/icon/placeholder.svg'
 import deleteIcon from '../../../asset/icon/delete.svg'
 
 import HandleValue from '../../../../stories/util/handle-value'
+import ProviderName from '../provider-name'
```
Comment (webholics) [inline src/app/component/model-list/model-list.story.js]: component imports should be put together

File: src/app/component/provider-name/_provider-name.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.ProviderName {
+  font-weight: $font-weight-bold;
```
Comment (webholics) [inline src/app/component/provider-name/_provider-name.scss]: should we enforce the grey text color here?

File: src/app/container/material-partial.js [hunk 5]
```diff
@@ -64,12 +63,13 @@ import ColorCardList from '../component/color-card-list'
...
 import RecommendedOfferSection from '../component/recommended-offer-section'
 import OfferCard from '../component/offer-card'
-import ProviderImage from '../component/provider-image'
+
 import OfferList from '../component/offer-list'
 import OfferItem from '../component/offer-item'
 
 import fastestIcon from '../../asset/icon/fastest.svg'
 import cheapestIcon from '../../asset/icon/cheapest.svg'
+import ProviderName from '../component/provider-name'
```
Comment (webholics) [inline src/app/container/material-partial.js]: component imports should be put together

File: src/app/component/cart-model-item/cart-model-item.js (hunk 1 above)
Comment (dreampulse) [reply src/app/component/cart-model-item/cart-model-item.js]: Why enforcing a react element here?
https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

**Pull request merged**